<?xml version="1.0"?>
<launch>
  <arg name="tool_name" default="tool"/> <!-- Name of the tool we are going to use -->
  
  <!-- Parameters to pass to the ROS node -->
  <arg name="move_group" default="manipulator"/>
  <arg name="ee_link" default="iiwa_link_ee"/>   <!-- tool_link_ee -->
  <arg name="ros_rate" default="0.1"/>
  
  <param name="/iiwa/toolName" type="string" value="$(arg tool_name)" /> <!--Sunrise tool-->
  
      <!-- start the realsense camera -->
  <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
  </include>


  <!--  This loads the robot description  -->
  <include file="$(find iiwa_moveit)/launch/moveit_planning_execution.launch">
    <arg name="sim" value="false"/>
  </include>



  <node ns="iiwa" name="initialize_pose" pkg="my_pcl_tutorial" type="initialize_object_template_iiwa_pose" respawn="false" output="screen" launch-prefix="gnome-terminal --command" >
    <param name="move_group" value="$(arg move_group)"/>
    <param name="ee_link" value="$(arg ee_link)"/>
    <param name="ros_rate" value="$(arg ros_rate)"/>
  </node>


  
  
</launch>